AUTHORS

```diff
@@ -619,8 +667,12 @@ Steven Merrill <steven.merrill@gmail.com>
 sudosurootdev <sudosurootdev@gmail.com>
```

Comment:
not signed
https://github.com/docker/docker/commit/2a1181f404387f6d530a86a6196b40a6dd541c20